src/crypto.js

```diff
@@ -14,15 +16,16 @@ export function stretchPassphrase (passphrase: string | Buffer, { salt, n, r, p
   return { key, salt }
 }
 
-export function aesEncrypt (key: Buffer, message: Buffer, opts = { cipher: 'aes-256-gcm', iv: crypto.randomBytes(IV_LEN_BYTES) }) : Object {
-  const cipher = crypto.createCipheriv(opts.cipher, key, opts.iv)
+export function aesEncrypt (key: Buffer, message: Buffer) : Object {
```

Comment:
will need to be a major version bump (note to self)

Reply:
This is not a documented public method; no need for major.